src/components/header.tsx

```diff
@@ -36,9 +38,7 @@ export const Header = () => {
           <Container>
             <HeaderLink to={Routes.HISTORY}>History</HeaderLink>
             <HeaderLink to={Routes.FAVORITES}>Favorites</HeaderLink>
-            <Button icon={IconBack} onClick={handleClick(Routes.HOME)}>
-              Log out
-            </Button>
+            <Button icon={IconBack} onClick={handleClick(Routes.HOME)}></Button>
```

Comment:
```suggestion
            <Button icon={IconBack} onClick={handleClick(Routes.HOME)} />
```
поставь правило линта на это